### PR TITLE
docs: Changing page to folder index.

### PR DESCRIPTION
**Pull request by zbwright**
Previous content on this page was redundant with worker-nodes.md. Changing this page to an index for the folder to allow for proliferation of troubleshooting guides within it.

### CODE REVIEW

**Comment by robszumski** [comment]
Note that this change will need a redirect if no index page gets published at this location.

**Comment by zbwright** [comment]
@robszumski The file's name did not change, only its content. We're keeping the file for later use. - was that your concern?

**Comment by robszumski** [comment]
Ah yes, I misunderstood

**Comment by zbwright** [comment]
Added descriptive text for each page - @joshix please approve.

File: Documentation/troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -1,68 +1,7 @@
...
-$ ssh core@192.0.2.18
-```
-
-View logs on the worker node by using `journalctl -xe` or similar tools. [Reading the system log][journalctl] has more information.
-
-
-[journalctl]: https://github.com/coreos/docs/blob/master/os/reading-the-system-log.md
+* [Troubleshooting Tectonic Installer](installer-terraform.md) describes troubleshooting the installation process itself, including reapplying after a partially completed run, and errors related to missing Terraform modules and failed `tectonic.service` unit.
+* [Disaster recovery of Scheduler and Controller Manager pods](controller-recovery.md) describes how to recover a Kubernetes cluster from the failure of certain control plane components.
+* [Troubleshooting worker nodes using SSH](worker-nodes.md) describes how to SSH into a controller or worker node to troubleshoot the pod.
```
Comment (joshix) [inline Documentation/troubleshooting/troubleshooting.md]: "...to troubleshoot at the host level."

File: Documentation/troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -1,68 +1,7 @@
...
-```bash
-# From master node
-$ ssh core@192.0.2.18
-```
-
-View logs on the worker node by using `journalctl -xe` or similar tools. [Reading the system log][journalctl] has more information.
-
-
-[journalctl]: https://github.com/coreos/docs/blob/master/os/reading-the-system-log.md
+* [Troubleshooting Tectonic Installer](installer-terraform.md) describes troubleshooting the installation process itself, including reapplying after a partially completed run, and errors related to missing Terraform modules and failed `tectonic.service` unit.
```
Comment (joshix) [inline Documentation/troubleshooting/troubleshooting.md]: These should be reference links as specified in the style guide

**Pull request merged**